### ISSUE TITLE
Enable Mosh

### ISSUE THREAD

**Issue by rking** [rondale-sc/remote_pair_chef#24]
http://mosh.mit.edu/#faq -
Q: Does mosh work with Amazon EC2?
    Yes, it works great, but please remember to open up UDP ports 60000–61000 on the EC2 firewall.

I'm not sure if this is easy to automate or what, but it'd definitely be nice to have.

**Comment by rwjblue** [comment]
I completely agree. I'll take a look at the [Opscode mosh cookbook](http://community.opscode.com/cookbooks/mosh).

The thing I am unsure of is how mosh honors the initial command from authorized_keys. Will review and see about adding a PR.

**Comment by rondale-sc** [comment]
rjackson have you gotten a chance to look at this?  Wouldn't mind pairing if you find yourself in need.

**Comment by rwjblue** [comment]
Yeah, sorry. Have a working copy locally. Will push a PR in a few...

Robert Jackson

-- twitter: rwjblue
-- github: rjackson
